api/prisma/migrations/20230309162534_add_affiliations_status/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Publication" ADD COLUMN     "affiliationsStatus" BOOLEAN DEFAULT TRUE;
```

Comment:
Please provide better names for these two columns, a boolean should start with is, has, should, and so on... Using these names to check something will make the code hard to read/understand.

Reply:
I'm sorry, but i don't agree with this. If something was poorly named by others, that doesn't mean we need to continue in the same manner just to be "consistent". Being consistent with wrong stuff it's not a good thing.